test/client_service_base.test.js

```diff
@@ -35,17 +38,18 @@ describe('Class ClientServiceBase', () => {
           'scalar.ledger.client.cert_holder_id': 'hold',
         };
         assert.throws(() => {
-          new ClientServiceBase(ledgerClient, protobuf, clientProperties);
+          new ClientServiceBase(service, protobuf, clientProperties);
         }, IllegalArgumentError, 'cert_pem');
       });
       it('when holder id is missing', () => {
         const clientProperties = {
           'scalar.ledger.client.private_key_pem': 'key',
           'scalar.ledger.client.cert_pem': 'cert',
+          'scalar.ledger.client.server_host': 'host',
           // 'scalar.ledger.client.cert_holder_id': 'hold',
```

Comment:
Do we want to keep it ?

Reply:
Sorry, since it is not needed, i'll remove it.